src/CPPDataMember.cxx

```diff
@@ -176,6 +177,7 @@ static CPPDataMember* pp_new(PyTypeObject* pytype, PyObject*, PyObject*)
     pyprop->fConverter      = nullptr;
     pyprop->fEnclosingScope = 0;
     pyprop->fDescription    = nullptr;
+    pyprop->fDoc            = PyUnicode_FromString("");
```

Comment:
`CPPDataMember` is the only one to have [direct access](https://github.com/wlav/CPyCppyy/pull/1/files#diff-32dc08abe156fbe8ad6252c1210ad1c4a044c640420f94e781dece2d744d354aR197) to the doc field/member instead of having some customized behaviour via a getter. I'm not sure how Python handles C `nullptr` vs Python `None`, so I wanted to stay on the safe side. But maybe we could replace `T_OBJECT_EX` by `T_OBJECT` and then leave it at `nullptr` initially, depending on how well this plays with Python's `help`.

Reply:
fix ind 3d5e7471f6139a7d40154500ec62df73d21cfd72

Reply:
The empty string is not Py_None, so that's not the comparison. Point is rather that if any other code wants to distinguish between "set" and "not set", then some Python string v.s `nullptr` is an easy way to distinguish.

Reply:
I was mostly unsure how Python reacts to the `__doc__` field being `nullptr`, i.e. whether that is interpreted as `None`, the empty string, or raises an exception. But it seems to work with setting it to `nullptr` initially. So is it alright the way it is now?